File: M1_1.java
public class M1_1 {
    public static void main(String[] args) throws Exception {

        int n = 1000; //escolhe a quantidade
        int tempoTrabalhoMs = 10;

        System.out.println("modelo 1:1");
        System.out.println("threads: " + n);
        Thread[] threads = new Thread[n];
        long inicio = System.nanoTime(); //começo
        //cria e inicia as threads
        for (int i = 0; i < n; i++) {
            threads[i] = new Thread(() -> {
                try { Thread.sleep(tempoTrabalhoMs); }
                catch (InterruptedException e) { Thread.currentThread().interrupt(); }
            });
            threads[i].start();
        }
        for (int i = 0; i < n; i++) threads[i].join();
        long fim = System.nanoTime(); //
        long tempoTotalMs = (fim - inicio) / 1_000_000;

        System.out.println("Tempo total: " + tempoTotalMs + " ms");
    }
}

File: M1_1.py
import time
import threading

def main():
    n = 10  #escolhe aqui tb
    tempo_trabalho = 0.01
  
    print("modelo 1:1")
    print("threads reais", n)
    threads = []
    inicio = time.time()#comeco
    #cria e inicia todas as threads reais
    for _ in range(n):
        t = threading.Thread(target=time.sleep, args=(tempo_trabalho,))
        threads.append(t)
        t.start()
    #espera todas terminarem
    for t in threads:
        t.join()

    fim = time.time()  #fim
    tempo_total_ms = (fim - inicio) * 1000
    print("tempo total: ", tempo_total_ms,"ms")

if __name__ == "__main__":
    main()
